libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -1419,7 +1419,9 @@ void AP_BLHeli::read_telemetry_packet(void)
     last_telem[last_telem_esc].count++;
 
     AP_Logger *logger = AP_Logger::get_singleton();
-    if (logger && logger->logging_enabled()) {
+    if (logger && logger->logging_enabled()
+        // log at 10Hz
+        && last_log_ms[last_telem_esc] < td.timestamp_ms - 100) {
```

Comment:
needs to be:
last_log_ms[last_telem_esc] - td.timestamp_ms > 100

Reply:
Fixed (although I think its the other way round)